src/Form/Input.js

```diff
@@ -241,7 +241,7 @@ class Input extends Component {
       ...rest
     } = this.props;
 
-    const isFloating = (floating && value && value.length > 0) || (floating && this.state.value);
+    const isFloating = (floating && value && value.length > 0) || (floating && this.state.value !== undefined);
```

Comment:
Should we account for `null` as well?

Reply:
It couldn't hurt, what's the use case for `null`?

Reply:
Rather than check all the falsy cases, let's just stick with the truthy case, and add a check for zero, since it's the only falsy case we want to include.